lib/ui/imagedecoratedstructs.flow

```diff
@@ -54,8 +55,10 @@ export {
 		mouseZoom : bool,
 		extraZoom : bool,
 		closeButtonAlign : int,
-		closeButtonFn : (() -> void) -> Form
+		closeButtonFn : (() -> void) -> Form,
+		zoomButtonM : Maybe<ZoomButtonDescription>
```

Comment:
I would suggest to avoid Maybe here, create global constant defaultZoomButton and pass here if there is no custom

Reply:
The problem is that standard zoom don't use button but cursor, so if we use default button stadard zoom behaviour will change.